Lib/test/test_pyrepl/test_interact.py

```diff
@@ -105,7 +105,7 @@ def test_runsource_shows_syntax_error_for_failed_compilation(self):
 
     def test_no_active_future(self):
         console = InteractiveColoredConsole()
-        source = "x: int = 1; print(__annotations__)"
+        source = "x: int = 1; print(__annotate__(1))"
```

Comment:
Why does printing `__annotations__` not work here? Shouldn't it call `__annotate__` under the hood anyway?

Reply:
Printing a global doesn't go through the descriptor protocol, so printing `__annotations__` will just throw a NameError; it is added to the namespace only when the `__annotate__` descriptor is called.